### PR TITLE
DOC: chat-ops

### LINKED ISSUE #38726: DOC: chat-ops
xref https://github.com/pandas-dev/pandas/pull/38444#issuecomment-749711369

There are some GitHub Actions workflows which can be triggered via comments:

1. If you comment `@github-actions pre-commit` on a pull request, it will trigger a workflow which will autofix formatting errors (e.g. from `black` or `isort`)
2. `@meeseeksdev backport ` will trigger a workflow which will backport a given change to a branch (e.g. `@meeseeksdev backport 1.2.x`)
3. If you comment `take` on an issue, it will trigger a workflow which will assign the issue to you

These should be documented somewhere in the docs - I think `doc/source/development/contributing.rst` might be a sensible location. `take` is already documented in there, but the other two aren't

### PR DESCRIPTION
**Pull request by 1nF0rmed**
- [x] closes #38726
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] Added documentation for `@githubactions pre-commit`
- [x] Added documentation for `@meeseeksdev backport`

### CODE REVIEW

**Comment by 1nF0rmed** [comment]
@githubactions pre-commit

File: doc/source/development/contributing.rst [hunk 1]
```diff
@@ -1610,6 +1610,28 @@ request by pushing to the branch on GitHub::
...
+Backporting (optional)
+----------------------
+
+In the case you want to apply changes to a stable branch from a newer branch then you
+can comment::
+
+    @meeseeksdev backport
+
+This will trigger a workflow which will backport a given change to a branch
+(e.g. ``@meeseeksdev backport 1.2.x``)
```
Comment (rhshadrach) [inline doc/source/development/contributing.rst:1633]: It's unclear here what `@meeseeksdev backport` will do without a version.

**Review by rhshadrach — CHANGES_REQUESTED** [review]
Thanks for the PR! I assume `meeseeksdev` can only be used by someone with write permissions; remarks should be made to that effect. I'm also wondering if there should be some language around backporting to the effect of "you shouldn't be using this unless you really know otherwise". Thoughts here @simonjayhawkins?

File: doc/source/development/contributing.rst [hunk 2]
```diff
@@ -1610,6 +1610,28 @@ request by pushing to the branch on GitHub::
...
+
+We use several styling checks(e.g. ``black``, ``flake8``, ``isort``) which are run after
+you make a pull request. If there is a scenario where any of these checks fail then you
+can comment::
+
+    @githubactions pre-commit
+
+on that pull request. This will trigger a workflow which will autofix formatting errors.
+
+Backporting (optional)
```
Comment (jreback) [inline doc/source/development/contributing.rst:1624]: this should only be in the maintainers section (i think we have one?)

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/development/contributing.rst [hunk 3]
```diff
@@ -1610,6 +1610,28 @@ request by pushing to the branch on GitHub::
...
     git push origin shiny-new-feature
 
+Autofixing formatting errors
+----------------------------
+
+We use several styling checks(e.g. ``black``, ``flake8``, ``isort``) which are run after
+you make a pull request. If there is a scenario where any of these checks fail then you
+can comment::
+
+    @githubactions pre-commit
```
Comment (MarcoGorelli) [inline doc/source/development/contributing.rst:1620]: this should be
```suggestion
    @github-actions pre-commit
```
, I missed out the `-` in the issue

File: doc/source/development/contributing.rst (hunk 2 above)
Comment (MarcoGorelli) [reply doc/source/development/contributing.rst:1624]: Yes, doc/source/development/maintaining.rst - @1nF0rmed could you move this over there please?

**Comment by 1nF0rmed** [comment]
I'll make these changes and commit ASAP

**Comment by MarcoGorelli** [comment]
> ASAP

:rofl: thanks, but there's no hurry

File: doc/source/development/contributing.rst (hunk 2 above)
Comment (1nF0rmed) [reply doc/source/development/contributing.rst:1624]: Does that include the backporting section?

Comment (MarcoGorelli) [reply doc/source/development/contributing.rst:1624]: yes, the whole part about `@meeseeksdev backport`

File: doc/source/development/maintaining.rst [hunk 4]
```diff
@@ -151,6 +151,28 @@ Here are some things to check when reviewing a pull request.
...
+Autofixing formatting errors
+----------------------------
+
+We use several styling checks (e.g. ``black``, ``flake8``, ``isort``) which are run after
+you make a pull request. If there is a scenario where any of these checks fail then you
+can comment::
+
+    @github-actions pre-commit
+
+on that pull request. This will trigger a workflow which will autofix formatting errors.
```
Comment (MarcoGorelli) [inline doc/source/development/maintaining.rst:163]: Sorry for being unclear - this part can go in `contributing.rst`, it's the backporting section that should go in `maintaining.rst`

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @1nF0rmed 

if any more comments @rhshadrach @MarcoGorelli

**Review by rhshadrach — APPROVED** [review]

**Comment by jreback** [comment]
thanks @1nF0rmed

**Pull request merged**